_pages/03__guidelines.md

```diff
@@ -144,4 +144,6 @@ As our main display typeface, [Poppins](https://fonts.google.com/specimen/Poppin
 There aren't really strict guidelines on how to use these typefaces—designing with them is up to your discretion. Likewise, there's no need to adhere to these type families, as they may change in the future. That being said, when in doubt and don't care about font specifics, use Poppins.
 
 {% include "assetbox-template-large.html", title: "Download Template" , file: "partials/large-partial-01.svg" %}
-{% include "assetbox-template-large.html", title: "Download Template" , file: "partials/large-partial-02.png" %}
\ No newline at end of file
+{% include "assetbox-template-large.html", title: "Download Template" , file: "partials/large-partial-02.png" %}
+
+{% include "color-grid-committees.html" %}
```

Comment:
This will break without a committee object passed through, but we can fix this once we start work on the guidelines page